src/keyboard-shortcuts.tsx

```diff
@@ -11,20 +9,21 @@ interface IProps {
   shortcuts: { [chord: string]: () => void } | Shortcut[];
 }
 
-@observer
 export class KeyboardShortcuts extends Component<IProps> {
-  shortcuts: {};
-  @observable layer: ShortcutLayer;
+  shortcuts: Shortcut;
```

Comment:
I'm not sure, but it seems like the type here would be:

```ts
shortcuts: Record<string, Shortcut> | Shortcut[];
```

That being said... Is this ever read, or only written?

Reply:
actually, it is neither read to or written to. Removed it